code/modules/jobs/job_types/chief_medical_officer.dm

```diff
@@ -26,6 +26,16 @@
 	display_order = JOB_DISPLAY_ORDER_CHIEF_MEDICAL_OFFICER
 	bounty_types = CIV_JOB_MED
 
+	mail_goodies = list(
+		/obj/item/scalpel/alien = 5,
+		/obj/item/hemostat/alien = 5,
+		/obj/item/retractor/alien = 5,
+		/obj/item/circular_saw/alien = 5,
+		/obj/item/surgicaldrill/alien = 5,
+		/obj/item/cautery/alien = 5,
+		/obj/effect/spawner/lootdrop/organ_spawner = 3,
+		/obj/effect/spawner/lootdrop/memeorgans = 3
```

Comment:
I am slightly offended medical doctors can't get the meme organs. Also the alien tools look pretty guaranteed, only one is needed for all the station to get alien tech